hugo.yaml

```diff
@@ -5,7 +5,6 @@ buildFuture: true
 # cSpell:ignore goldmark github hugo readingtime docsy subdir lastmod pygments linenos catmullrom norsk gu
 
 # Language settings
-contentDir: content
 defaultContentLanguage: en
 defaultContentLanguageInSubdir: false
```

Comment:
I think if this is `false` then we don't need to put the English content in `en/`. I would prefer not to have to remember to put the English content in `en/`, so what if we try not moving it and leaving this as `false`? Let me try that in a separate branch and see what happens. https://gohugo.io/configuration/languages/#base-settings

Reply:
Well I never would have predicted this, but when you set `defaultContentLanguageInSubdir: true`, then `en/` gets added to the English URLs! For example, locally the home page is http://localhost:1313/en/. So I guess I am OK with approving this PR for now.